sys/usb/usbus/usbus.c

```diff
@@ -121,6 +121,17 @@ uint16_t usbus_add_interface(usbus_t *usbus, usbus_interface_t *iface)
     return idx;
 }
 
+void usbus_add_interface_alt(usbus_t *usbus, usbus_interface_t *iface,
```

Comment:
Is there a reason for including `usbus_t` in the parameters here? It is not used in the function.

Reply:
There is no specific reason. I just copy-paste the add_interface fonction and change the bare minimum.
Should I drop this parameter ?